acmwebsite/lib/helpers.py

```diff
@@ -26,7 +26,11 @@ def markdown(*args, strip_par=False, **kwargs):
 
 
 def icon(icon_name):
-    return Markup('<i class="glyphicon glyphicon-%s"></i>' % icon_name)
+    return Markup('<i class="glyphicon glyphicon-{}"></i>'.format(icon_name))
```

Comment:
Glyphicons are gone in Bootstrap 4... is this ever called anyway?

Reply:
We use it in `acmwebsite/templates/survey_results.xhtml` to display the up and down arrows on the survey results page. We can easily replace those though. Added a note on #67 about this.